FurEver/Pages/DogQuiz.cshtml

```diff
@@ -0,0 +1,50 @@
+﻿@page
+@model FurEver.Pages.DogQuizModel
+@{
+    ViewData["Title"] = "Dog Breed Quiz";
+}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <title>@ViewData["Title"]</title>
+    <link rel="stylesheet" href="~/css/site.css" />
+    <link rel="stylesheet" href="~/css/quiz.css" />
+</head>
+<body>
+    <div class="container">
+        <h1>Dog Breed Quiz</h1>
+        @if (Model.QuizFinished)
+        {
+            <h2>Your Score: @Model.Score / @Model.TotalQuestions</h2>
+            <a href="/DogQuiz" class="btn">Start Again</a>
+            <!-- Link to navigate back to the home page -->
+            <a href="/" class="btn">Back to Home</a> <!-- This is the link to the home page -->
+        }
+        else if (Model.QuizImageUrl != null)
+        {
+            <div class="dog-image">
+                <img src="@Model.QuizImageUrl" alt="Guess the Dog Breed" />
+            </div>
+            <form method="post">
+                <p>Question @Model.CurrentQuestionNumber of @Model.TotalQuestions: Select the breed:</p>
+                @foreach (var option in Model.BreedOptions)
+                {
+                    <div>
+                        <input type="radio" name="selectedBreed" value="@option" required /> @option
+                    </div>
+                }
+                <button type="submit" class="btn">Submit Answer</button>
+            </form>
+        }
+
+        @if (Model.AnswerResult != null)
+        {
+            <p class="@((Model.AnswerResult.Contains("Oops!") ? "quiz-result-wrong" : "quiz-result"))">
```

Comment:
# Technical Change #2 (line 44):
## Allowing for an incorrect quiz answer to be shown in red instead of green
This change reduces technical debt by improving the accessibility and clarity of the user-interface, aligning it with WCAG 2.1 guideline 1.4.1 Use of Color. By allowing incorrect quiz answers to be displayed in red instead of green, users can easily distinguish between correct and incorrect responses without relying solely on color differentiation, which can be problematic for individuals with color vision deficiencies. This modification ensures that feedback is more universally understood and accessible to all users, enhancing the overall user experience. Proactively addressing accessibility needs minimizes the risk of future rework to make the application compliant with accessibility standards. Moreover, this change reduces user confusion and potential errors, making the quiz interaction more intuitive and robust, ultimately leading to a more maintainable and user-friendly codebase. By implementing this enhancement, you also ensure better compliance with accessibility requirements, decreasing the likelihood of needing extensive accessibility audits and fixes in the future.
Link to Reference: https://www.w3.org/WAI/WCAG22/Understanding/use-of-color.html